pypicloud/cache/base.py

```diff
@@ -6,6 +6,7 @@
 from pyramid.settings import asbool
 
 import posixpath
+from flywheel.fields.types import UTC
```

Comment:
Ah, yes, makes perfect sense. Maybe we should be using pytz instead? Not a bad dep to have, and reduces such boilerplate. I'm open to both approaches, let me know which one you prefer.